.github/pull_request_template.md

```diff
@@ -51,4 +51,5 @@ Delete this section if no tips.
 ### Checklist
 
 - [ ] This PR has adequate test coverage / QA involvement has been duly considered.
-- [ ] This PR adds a release note for any [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/user/content/release-notes.md#what-changes-require-a-release-note).
+- [ ] This PR includes the following [user-facing behavior changes](/doc/developer/guide-changes.md#what-changes-require-a-release-note):
+  - <!-- Add release notes here or explicitly state that there are no user-facing behavior changes. -->
```

Comment:
Maybe make this into its own H3 section? Sounds easier to copy and paste later.

Reply:
Done.